de_web_plugin.cpp

```diff
@@ -716,6 +716,10 @@ void DeRestPluginPrivate::apsdeDataIndication(const deCONZ::ApsDataIndication &i
                     {
                         sensorNode = getSensorNodeForAddressAndEndpoint(ind.srcAddress(), 0x03);
                     }
+                    else if (sensorNode->modelId().contains(QLatin1String("86opcn01"))) //Aqara Opple enable events from all multistate clusters
```

Comment:
The model ends with "86opcn01" in all cases, right? Perhaps I'm a performance nut, but it would seem more efficient to use "endsWith" rather than "contains", since presumably endsWith compares starting at the end of the string, and would fail immediately if the tail doesn't match.

Reply:
Not sure if this will make a significant difference...

Reply:
I think it just offends me to use a general substring search when we know very well that we only care about the suffix match. It’s useful to consider that this substring search occurs for every model, on every iteration of this code, and only succeeds for these opple switches. Consider how many failing string searches that implies...! Conversely, with the suffix match (endsWith), in most cases the common failure case occurs after looking at one character. 

This comment applies to all of the uses of contains in this PR.